crates/relayer/src/testdata/test_update_eth_client/case-2/headers_part_2.json

```diff
@@ -0,0 +1,220 @@
+    "proposer_index": "379737",
+    "parent_root": "0xde75948a5d270764c811ad7c6ad4194cc838fd4854d27e77db10188d1c4afdf7",
+    "state_root": "0xc9f3aae4892e0dcdee5c3996a30014046f43b67c6b5282d440aa1d6ee7b4f4f3",
+    "body_root": "0x8b31be752cd461879c8d494bbfb912e4e98c94d40a72d1b466ba42c576c28473"
+  },
+  {
+    "slot": "5601240",
+    "proposer_index": "335846",
+    "parent_root": "0x2ed956585fdff73499aae11ce0cd5c5a6e8f571a5165ceba5294cfeff5f3db65",
+    "state_root": "0xd39f400af396b3bbf995ec0e87a92de0d2489c9bf4011adf18c19736823be286",
+    "body_root": "0x49813ac44c0fd281b3428ea895bac5f3e224d1845e331b2406594fc0b41b5ee1"
+  },
+  {
+    "slot": "5601241",
+    "proposer_index": "264112",
+    "parent_root": "0x098be0120acd31fbe3360be6966aa15a1c0c34b056738884bbceb75c7817ed93",
+    "state_root": "0x3f1fa4e547a561aa4da3d15671fc62ed2b5a4ac19e6847903f9799b5e13cbc39",
+    "body_root": "0x6acab0963b37b4d9fde287370185c19a6d0571facc1f82b7279b4fef30e91227"
+  },
+  {
+    "slot": "5601242",
+    "proposer_index": "388149",
+    "parent_root": "0xe5d4bd0e7e126233f9f1eec853c6b887da4b32cf190ecd64aee0babf651308ba",
+    "state_root": "0x2e22060d8a9763ccda6bffba1275e15c516a6944b5a33a7dda261bf1d1285e9e",
+    "body_root": "0xa0e37d5a82c78a21bda6d5ed0e6de8ff8b5b706cd707440738233e96a33aff87"
+  },
+  {
+    "slot": "5601243",
+    "proposer_index": "430151",
+    "parent_root": "0x201fb138d1b25d8cfdee3fed6bb1556afa745c69ff4a5c25d7314ffe26a89129",
+    "state_root": "0xd5f303723c5e5d07e4934dfccb9eabc715cf18823a0a759b999de5fdd0456e38",
+    "body_root": "0x3ab4bc8f349df6e0d45a2662d9d9091d53ee4a7a2add5941758b5042d1ae4617"
+  },
+  {
+    "slot": "5601244",
+    "proposer_index": "398396",
+    "parent_root": "0xdef5228a43f133b52e15ef019de96f070660128c7ec447cd79ffb0eb5d4c91a4",
+    "state_root": "0x4ea50b528a1702f5a3ac62bad6ef3ba447c168c483ac9cb8d2ddb5c5da382fba",
+    "body_root": "0x042441fef8d2fb2cda757a1600561f19ab989f49b1a7196f3cb142d30cd3c273"
+  },
+  {
+    "slot": "5601245",
+    "proposer_index": "410870",
+    "parent_root": "0xdc9a3265c7852b152b012e08075d475ec75185350727aefac2d44b85f586dcd9",
+    "state_root": "0x719c7980654d8f77c5d8dee1dbe2af3703478491aa772cb00bab904fbbaf03e7",
+    "body_root": "0x7be96b68552d2d4e9b4487210615c2955d8673eaee33f13f1a71b7dacd33178a"
+  },
+  {
+    "slot": "5601246",
+    "proposer_index": "434396",
+    "parent_root": "0xf64632d6b60f2a78089baf71249a3b61f3a880b0a2224ddb3908e35527cd59c7",
+    "state_root": "0x5673072cd48be64834bedba00c8939c2b291cc0219f8a57976e653c3ce5c36e9",
+    "body_root": "0xb62eec039c1937deeec8865cfd228677c26c783d937f24f322a38f74b6690710"
+  },
+  {
+    "slot": "5601247",
+    "proposer_index": "127500",
+    "parent_root": "0x2f5ea1d1eac40a94551529ad896a916bce502cc4619060a5d8888b3de6ded336",
+    "state_root": "0x21f64ceb3418e61426285c707de204a854159e0ea231476b136a74b82a061eee",
+    "body_root": "0x9a57242cb1adc4f6d15cb654b9fa16a80f3df95df08dbaaae3aecae877f2eb81"
+  },
+  null,
```

Comment:
In this testcase, the beacon block for slot 5601248 is null; then we create an empty beacon block header for MMR.